daphne/src/lib.rs

```diff
@@ -755,11 +737,9 @@ impl DapTaskConfig {
     /// Leader: Resolve taskprov advertisement to send in a request to the Helper.
     pub(crate) fn resolve_taskprove_advertisement(&self) -> Result<Option<String>, DapError> {
         if let DapTaskConfigMethod::Taskprov { info } = &self.method {
-            if info.is_none() && self.version != DapVersion::Draft02 {
+            if info.is_none() {
                 // The task config indicates that the configuration method was taskprov, but we
-                // don't have enough information to construct the advertisement. This is not a
-                // problem for draft02, however, because the task config was encoded by the report
-                // extensions in that version.
+                // don't have enough information to construct the advertisement.
```

Comment:
Is there a situation in which `info` being `None` makes sense? if not maybe it's type shouldn't be `Option`

Reply:
Alright, defer this to an issue. I might pick it up

Reply:
Done.